Pipfile

```diff
@@ -6,12 +6,16 @@ name = "pypi"
 [packages]
 python-dotenv = "~=1.0"
 requests = "~=2.26"
-ruff = "~=0.7"
 tqdm = "~=4.66"
 
 [dev-packages]
 pytest = "~=8.3"
 pytest-cov = "~=5.0"
+ruff = "~=0.7"
 
 [requires]
 python_version = "3.12"
+
+[scripts]
+dev = "pipenv run python -m mediabridge.main"
```

Comment:
I think that `main` would be a better name. Just looking through the README changes, it's a weird sentence: "pipenv run dev runs the main script".

Reply:
I figured eventually we'd add a "prod" script, so I didn't want to name the dev script "main" since I feel the difference between "main" and "prod" is more ambiguous than "dev" and "prod".